native/src/components/PinchableBox.tsx

```diff
@@ -0,0 +1,319 @@
+import React, { ReactElement } from 'react'
```

Comment:
I think there is no need for review. Just test it.

Reply:
a little hint how to test it would be nice :)

Reply:
It is used when you click on images in content.